src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -633,12 +645,154 @@ public Snapshot makeSnapshotFromSnapshotRequest(SnapshotRequestModel snapshotReq
         .tags(TagUtils.sanitizeTags(snapshotRequestModel.getTags()));
   }
 
-  private void createSnapshotTablesFromDatasetAsset(
-      String datasetName, String assetName, SnapshotSource snapshotSource, Snapshot snapshot) {
-    Dataset queryDataset = datasetService.retrieveByName(datasetName);
-    AssetSpecification queryAssetSpecification = getAssetByNameFromDataset(queryDataset, assetName);
-    snapshotSource.assetSpecification(queryAssetSpecification);
-    conjureSnapshotTablesFromAsset(snapshot, snapshotSource);
+  public SnapshotAccessRequestResponse getSnapshotAccessRequestById(UUID accessRequestId) {
```

Comment:
Does this need a helper method? I would say that `snapshotRequestDao.getById` would be clearer to me.